<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBSBasicScenes</class>
 <widget class="QWidget" name="OBSBasicScenes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>218</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="scenesFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="scenesLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="SceneTree" name="scenes">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::TargetMoveAction</enum>
        </property>
        <addaction name="actionRemoveScene"/>
       </widget>
      </item>
      <item>
       <widget class="QToolBar" name="scenesToolbar">
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="floatable">
         <bool>false</bool>
        </property>
        <addaction name="actionAddScene"/>
        <addaction name="actionRemoveScene"/>
        <addaction name="separator"/>
        <addaction name="actionSceneUp"/>
        <addaction name="actionSceneDown"/>
       </widget>
      </item>
      <item>
       <spacer name="scenesFixedSizeHSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionAddScene">
   <property name="icon">
    <iconset resource="obs.qrc">
     <normaloff>:/res/images/plus.svg</normaloff>:/res/images/plus.svg</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="themeID" stdset="0">
    <string notr="true">addIconSmall</string>
   </property>
  </action>
  <action name="actionRemoveScene">
   <property name="icon">
    <iconset resource="obs.qrc">
     <normaloff>:/res/images/minus.svg</normaloff>:/res/images/minus.svg</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
   <property name="themeID" stdset="0">
    <string notr="true">removeIconSmall</string>
   </property>
  </action>
  <action name="actionSceneUp">
   <property name="icon">
    <iconset resource="obs.qrc">
     <normaloff>:/res/images/up.svg</normaloff>:/res/images/up.svg</iconset>
   </property>
   <property name="text">
    <string>MoveUp</string>
   </property>
   <property name="themeID" stdset="0">
    <string notr="true">upArrowIconSmall</string>
   </property>
  </action>
  <action name="actionSceneDown">
   <property name="icon">
    <iconset resource="obs.qrc">
     <normaloff>:/res/images/down.svg</normaloff>:/res/images/down.svg</iconset>
   </property>
   <property name="text">
    <string>MoveDown</string>
   </property>
   <property name="themeID" stdset="0">
    <string notr="true">downArrowIconSmall</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SceneTree</class>
   <extends>QListWidget</extends>
   <header>scene-tree.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="obs.qrc"/>
 </resources>
 <connections/>
</ui>
